bindings/python/examples/.pylintrc

```diff
@@ -0,0 +1,8 @@
+[MESSAGES CONTROL]
+disable=missing-module-docstring,
```

Comment:
is the spacing supposed to be like this?

Reply:
no, it shouldn't. This is a weird behavior, because vscode shows it differently, Fixed